experimental/packages/exporter-logs-otlp-http/test/browser/OTLPLogExporter.test.ts

```diff
@@ -32,7 +32,7 @@ describe('OTLPLogExporter', () => {
     sinon.restore();
   });
 
-  if (typeof process === 'undefined') {
+  if (global.process?.versions?.node === undefined) {
```

Comment:
needed so that we mock the right thing for the right platform, `process` is now defined due to the required polyfill, so we're using `process?.versions?.node` instead to determine if we're running in Node.js or the browser.